src/tools/pub/package-layout.md

```diff
@@ -26,7 +26,7 @@ might look like:
 {% prettify none tag=pre+code %}
 enchilada/
   .dart_tool/ *
-  .packages *
+  package_config.json *
```

Comment:
The `package_config.json` file is part of the  `.dart_tool/` directory, so I don't think this is needed:

```suggestion
```